packages/rdma-core/logdog.rdma.conf

```
@@ -0,0 +1,3 @@
+exec ibv_devinfo.log ibv_devinfo
+glob /sys/class/infiniband/*/device/p2p
+glob /sys/class/infiniband/*/ports/1/hw_counters/*
```

Should we check all ports?
```suggestion
glob /sys/class/infiniband/*/ports/*/hw_counters/*
```

(also, please configure your editor to add a final newline to files so that this red symbol doesn't show)